<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordManager_por_JVG</class>
 <widget class="QMainWindow" name="PasswordManager_por_JVG">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>465</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>440</width>
    <height>465</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1500</width>
    <height>600</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Password Manager</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::TabShape::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="menu_password_manager">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pass_manager">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>30</red>
             <green>30</green>
             <blue>30</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>30</red>
             <green>30</green>
             <blue>30</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled/>
        </palette>
       </property>
       <attribute name="title">
        <string>Administrador de contraseñas</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="horizontalSpacing">
         <number>2</number>
        </property>
        <item row="6" column="1" colspan="5">
         <widget class="QLineEdit" name="campo_AgregarUsuario">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="15" column="0" colspan="6">
         <widget class="QLabel" name="label_7">
          <property name="styleSheet">
           <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
          <property name="text">
           <string>Contraseña generada: </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="13" column="0" colspan="6">
         <widget class="QPushButton" name="boton_GenerarPass">
          <property name="text">
           <string>Generar contraseña</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QLabel" name="texto_AgregarMail">
          <property name="text">
           <string>Correo:</string>
          </property>
         </widget>
        </item>
        <item row="14" column="0" colspan="6">
         <widget class="Line" name="line_2">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="1" colspan="5">
         <widget class="QLineEdit" name="campo_AgregarServicio">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="0" alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QLabel" name="texto_AgregarServicio">
          <property name="text">
           <string>Servicio:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1" colspan="4">
         <widget class="QLineEdit" name="campo_AgregarContra">
          <property name="focusPolicy">
           <enum>Qt::FocusPolicy::StrongFocus</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::EchoMode::Password</enum>
          </property>
         </widget>
        </item>
        <item row="10" column="0" colspan="6">
         <widget class="QLabel" name="texto_Generar">
          <property name="styleSheet">
           <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
          <property name="text">
           <string>Generar contraseña</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="9" column="0" colspan="6">
         <widget class="Line" name="line">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Sunken</enum>
          </property>
          <property name="lineWidth">
           <number>2</number>
          </property>
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="12" column="0" colspan="2">
         <widget class="QCheckBox" name="check_Generar_IncluirMayus">
          <property name="text">
           <string>Añadir mayusculas</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0" alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QLabel" name="texto_AgregarPass">
          <property name="text">
           <string>Contraseña:</string>
          </property>
         </widget>
        </item>
        <item row="16" column="0" colspan="4">
         <widget class="QLineEdit" name="campo_contraGen">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::EchoMode::Password</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="1" colspan="5">
         <widget class="QLineEdit" name="campo_AgregarMail">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="12" column="2" colspan="2" alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QCheckBox" name="check_Generar_IncluirNumeros">
          <property name="text">
           <string>Añadir numeros</string>
          </property>
         </widget>
        </item>
        <item row="17" column="0" colspan="6">
         <widget class="QPushButton" name="boton_copiarContraGen">
          <property name="text">
           <string>Copiar contraseña</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="6">
         <widget class="QLabel" name="texto_Agregar">
          <property name="styleSheet">
           <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
          <property name="text">
           <string>Añadir contraseña</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="12" column="5" alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QCheckBox" name="check_Generar_IncluirSimbolos">
          <property name="text">
           <string>Añadir símbolos (!@#$)</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0" colspan="2">
         <widget class="QLabel" name="texto_LongPass">
          <property name="text">
           <string>Longitud contraseña:</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0" colspan="6">
         <widget class="QPushButton" name="boton_AgregarDatosUser">
          <property name="text">
           <string>Guardar contraseña</string>
          </property>
         </widget>
        </item>
        <item row="11" column="2">
         <widget class="QLabel" name="slider_Addon_MostrarLongContraSeleccionada">
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="11" column="3" colspan="3">
         <widget class="QSlider" name="longitud_contra_gen">
          <property name="minimum">
           <number>8</number>
          </property>
          <property name="maximum">
           <number>32</number>
          </property>
          <property name="value">
           <number>32</number>
          </property>
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="7" column="5" alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QCheckBox" name="check_Agregar_MostrarContra">
          <property name="text">
           <string>Mostrar contraseña</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QLabel" name="texto_AgregarUser">
          <property name="text">
           <string>Usuario:</string>
          </property>
         </widget>
        </item>
        <item row="16" column="4" colspan="2" alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QCheckBox" name="check_MostrarContraGen">
          <property name="text">
           <string>Mostrar contraseña generada</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="6" alignment="Qt::AlignmentFlag::AlignHCenter">
         <widget class="QLabel" name="texto_agregarDatosInformacion">
          <property name="palette">
           <palette>
            <active>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Text">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>0</green>
                <blue>4</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="BrightText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>107</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="ButtonText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Text">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>0</green>
                <blue>4</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="BrightText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>107</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="ButtonText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="BrightText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>107</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="bd_pass">
       <attribute name="title">
        <string>Contraseñas guardadas</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" colspan="2">
         <widget class="QTableWidget" name="tabla_pass">
          <property name="contextMenuPolicy">
           <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
          </property>
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::DragDropMode::DragOnly</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>5</number>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>50</number>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Servicio</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Usuario</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Email</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Contraseña</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
